src/main/java/com/example/refreshablesecretmanager/WebController.java

```diff
@@ -19,11 +19,12 @@ public WebController(SecretProvider secretProvider, SecretManagerOperations oper
   @GetMapping("/getSecret")
   @ResponseBody
   public String getSecret() {
-    String secretId = secretManager.getSecretString(secretProvider.getSecret());
+   // String secretId = secretManager.getSecretString(secretProvider.getSecret());
+    String secret = secretProvider.getSecret();
```

Comment:
Secret is retrieved using `secretManager` rather than read from a `SecretProvider` object.